apps/tup-cms/src/taccsite_cms/settings_custom.py

```diff
@@ -205,6 +205,7 @@
 TACC_BLOG_CUSTOM_MEDIA_POST_CATEGORY = 'multimedia'
 TACC_BLOG_SHOW_ABSTRACT_TAG = 'external'
 
+TACC_BLOG_CATEGORY_ORDER = ['press-release', 'feature-story', 'multimedia', 'podcast']
```

Comment:
Just adds [what is removed from Core-CMS](https://github.com/TACC/Core-CMS/commit/a8316ca).